plugins/cef/src/parse.cpp

```diff
@@ -113,4 +114,27 @@ parse_extension(std::string_view extension) {
   return result;
 }
 
+caf::error convert(const message& msg, type& schema) {
+  static constexpr auto name = "cef.event";
+  // These fields are always present.
+  auto fields = std::vector<struct record_type::field>{
+    {"cef_version", count_type{}},     {"device_vendor", string_type{}},
+    {"device_product", string_type{}}, {"device_version", string_type{}},
+    {"signature_id", string_type{}},   {"name", string_type{}},
+    {"severity", string_type{}},
+  };
+  // Infer extension record, if present.
+  if (!msg.extension.empty()) {
+    std::vector<struct record_type::field> ext_fields;
+    ext_fields.reserve(msg.extension.size());
+    for (const auto& [key, value] : msg.extension) {
+      // FIXME: deduce type based on value.
```

Comment:
@dominiklohmann is there a way to call a function from `infer_command.cpp` that, given a string, computes the deduced value? Then this PR would be done aside from documentation.

Reply:
I don't exactly understand what "given a string" would mean. Best effort guess: You're looking for some combination of `to<data>(string)` and `type::infer(data)`.

Reply:
Yes, exactly! I pushed a commit that implements this.

Reply:
Note that `type::infer` can fail and give you an invalid type as a result, you need to check its result for truthiness. Other than that this looks correct to me.